android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -29,17 +30,17 @@
  * Date: 8/18/2015
  */
 public class LocationActivity extends BaseActivity implements SearchView.OnQueryTextListener {
+    private static final int locationWiseSessionList = 1;
     final private String SEARCH = "searchText";
-
+    @BindView(R.id.recyclerView_locations)
+    RecyclerView sessionRecyclerView;
+    @BindView(R.id.txt_no_sessions)
+    TextView noSessionsView;
+    @BindView(R.id.toolbar_locations)
+    Toolbar toolbar;
     private SessionsListAdapter sessionsListAdapter;
-
+    private GridLayoutManager gridLayoutManager;
     private List<Session> mSessions;
-    private static final int locationWiseSessionList = 1;
-
```

Comment:
dude you need to stop refactoring everything. It seems that this is an attempt to increase the number of lines of code modified. We will just close PR's where we see this kind of behaviour in the future.